raiden-network/light-client
# Notification panel foundation

**Thank you for submitting this pull request :)**

Fixes # 

**Short description**
Adds the foundation for a notification panel. A new modal with a separate routing which displays the notification panel.


**Definition of Done**


**Steps to manually test the change (dApp)**

1. Open the dApp and connect
2. When connected a notification icon will appear in the header from which you can access the notification panel.
